### PR TITLE
Improvements to attendance monitoring page

### PR DESCRIPTION
**Pull request by Jithmi004**

### CODE REVIEW

File: src/components/exam_attendance_table.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
```
Comment (Malith-19) [inline src/components/exam_attendance_table.js]: We don't need to import all from the react right?

File: src/pages/exam_attendance_report.js [hunk 2]
```diff
@@ -54,14 +57,27 @@ const Report = () => {
     //     }
     //     fetchData()
     // }, [])
-
+  //const theme = useTheme();
```
Comment (Malith-19) [inline src/pages/exam_attendance_report.js]: Let's remove the unused lines.

Comment (chathuranga-jayanath-99) [reply src/pages/exam_attendance_report.js:60]: okay

File: src/components/exam_attendance_table.js [hunk 3]
```diff
@@ -7,54 +8,134 @@ import TableHead from '@mui/material/TableHead';
...
+            </IconButton>
+        </Box>
+    );
+}
 
 const ExamAttendanceTable = (props) => {
     const rows = props.data;
     const [page, setpage] = useState(0);
+    const emptyRows = page > 0 ? Math.max(0, (1 + page) * rowperpage - rows.length) : 0;
+
```
Comment (chathuranga-jayanath-99) [inline src/components/exam_attendance_table.js]: we can remove this additional new line

**Review by Malith-19 — APPROVED** [review]

**Review by Malith-19 — APPROVED** [review]
All good other than the extra line @chathuranga-jayanath-99  mentioned.

**Review by chathuranga-jayanath-99 — APPROVED** [review]

**Pull request merged**